docs/book/src/developer/providers/cluster-infrastructure.md

```diff
@@ -7,6 +7,8 @@ Examples might include networking, load balancers, firewall rules, and so on.
 
 ## Data Types
 
```

Comment:
Do we want/ should have similar documentation for bootstrap and control plane templates?

Reply:
I think it makes sense to be consistent. I just hadn't considered non core bootstrap / cp templates when I wrote it.

Reply:
No doc exists for cp, follow up issue filed as #5393

Reply:
Thx